src/template.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
```

Comment:
- [ ] [_OPTIONAL_] It would be better to rename this file to **index.html** since it explicitly says to create index.html file.

![image](https://user-images.githubusercontent.com/59416136/165375361-23fbbccb-3d76-46b9-baf3-7dfd0e2b5724.png)